core/poetry.lock

```diff
@@ -168,100 +165,87 @@ pycparser = "*"
 
 [[package]]
 name = "charset-normalizer"
-version = "3.0.1"
+version = "3.1.0"
 description = "The Real First Universal Charset Detector. Open, modern and actively maintained alternative to Chardet."
 category = "main"
 optional = true
-python-versions = "*"
+python-versions = ">=3.7.0"
 files = [
-    {file = "charset-normalizer-3.0.1.tar.gz", hash = "sha256:ebea339af930f8ca5d7a699b921106c6e29c617fe9606fa7baa043c1cdae326f"},
```

Comment:
are all of these indeed related? or did we now pull in tons of upgraded dependencies? this would increase risk in RDI

Reply:
Yep. It just after upgrading to 3.8 and `poetry add fastparquet@2023.2.0`.